src/context/marriageOptions.ts

```diff
@@ -0,0 +1,19 @@
+import { createContext } from 'react';
+
+export interface MarriageOptionsProps {
```

Comment:
Might be worthwhile to use the `HuwelijksplannerState` which is already available in this project. If, for any reason, you think it's better to implement it this way, fine by me!